vividus-plugin-mobile-app/src/main/resources/properties/defaults/run.properties

```diff
@@ -0,0 +1,7 @@
+spring.profiles.active=mobile_app
+
+bdd.all-meta-filters=groovy: (!layout || layout.contains('mobile_app')) && !skip && (${bdd.meta-filters})
```

Comment:
I propose to do not make `mobile_app` as embedded vividus meta, let's allow users configure it